ui/apps/platform/src/Components/FilteredWorkflowViewSelector/useFilteredWorkflowViewURLState.ts

```diff
@@ -12,7 +12,14 @@ function useFilteredWorkflowViewURLState(): FilteredWorkflowViewURLStateResult {
         filteredWorkflowViews
     );
 
-    return { filteredWorkflowView, setFilteredWorkflowView };
+    const result = {
+        filteredWorkflowView,
+        setFilteredWorkflowView: (value: FilteredWorkflowView) => {
+            setFilteredWorkflowView(value);
+        },
+    };
```

Comment:
I didn't want to expose the `useURLStringUnion` setter and allow passing any string. This guards against non-filtered-workflow-view states from being passed as an argument.

Reply:
This is good thinking although I don't think it is totally necessary. The useURLStringUnion does validation internally and if an invalid string is passed it will ignore it as a no-op. If the use case here is to do something (other than nothing) when an invalid value is passed, I can see there being a benefit to enforce that at the caller.